spec/lib/local-links-manager/check_links/link_status_requester_spec.rb

```diff
@@ -0,0 +1,49 @@
+require 'rails_helper'
+require 'local-links-manager/check_links/link_status_requester'
```

Comment:
Local links manager doesn't seem to have autoloading. I would say out of the scope of this PR?